charger/ocpp.go

```diff
@@ -94,20 +94,35 @@ func NewOCPPFromConfig(other map[string]interface{}) (api.Charger, error) {
 		currentsG = c.currents
 	}
 
+	var voltagesG func() (float64, float64, float64, error)
+	if c.hasMeasurement(types.MeasurandVoltage + ".L3") {
+		voltagesG = c.voltages
+	}
+
 	var phasesS func(int) error
 	if c.phaseSwitching {
 		phasesS = c.phases1p3p
 	}
 
-	return decorateOCPP(c, powerG, totalEnergyG, currentsG, phasesS), nil
+	var socG func() (float64, error)
+	if c.hasMeasurement(types.MeasueandSoC) { // typo in ocpp-go
+		socG = c.soc
+	}
+
+	var currentG func() (float64, error)
+	if c.hasMeasurement(types.MeasurandCurrentOffered) {
+		currentG = c.getMaxCurrent
```

Comment:
Brauchen wir den? Bisher haben wir das immer nur ausgeprägt, wenn eine Box anderenfalls nicht vernünftig funktioniert.

Reply:
Hat hier (auch) eine andere Funktion und wird u.a. für Enabled() verwendet.

Muss daher bleiben und die anderen tun auch nicht weh an der Stelle.
Im Zweifelsfall tun sie hier einfach nichts und die Funktionen bleiben ohnehin undekoriert, da die automatische Erkennung hier noch fehlt, die diese abonnieren würde.

Reply:
Ok. Die anderen Messwerte gehören allerdings nicht in diesen PR.

Reply:
Ich verstehe es nicht. Enabled verwendet `c.getCurrent`. Hier geht es um `c.getMaxCurrent`?

Reply:
Die Baustelle jetzt für einzelne Messwertverarbeitungen jeweils einzeln aufzumachen bzw. zu splitten halte ich für nicht zielführend.
Das Thema ist mit diesem PR einfach dann vollumfänglich erledigt.
Sie tun im Zweifelsfall hier auch einfach nichts und insbesondere nicht weh.